staging/src/k8s.io/kubectl/pkg/cmd/completion/completion.go

```diff
@@ -167,7 +167,7 @@ func runCompletionBash(out io.Writer, boilerPlate string, kubectl *cobra.Command
 		return err
 	}
 
-	return kubectl.GenBashCompletionV2(out, false) // TODO: Upgrade to Cobra 1.3.0 or later before including descriptions (See https://github.com/spf13/cobra/pull/1509)
+	return kubectl.GenBashCompletionV2(out, false)
```

Comment:
This TODO comment means you can set the second arg to true now, so that descriptions are included in completion output.  But you will need to check it to make sure it is working.

Reply:
ok,thank you